cremalab/sarah-todo
# Adasp 31 redux

## Description

DESCRIPTION_HERE

### 🔗 Reference

> _Relevant links (e.g. issue, design, etc.)_

- [ADASP-31](https://crema.atlassian.net/browse/ADASP-31)

### 📋 Acceptance Criteria

> _Checked off by PR **Assignees**_

- [x] saves new task in reducer
- [x] deletes task in reducer
- [x] edit task status in reducer
- [x] edit task description in reducer





### 👩‍🔬 Test Instructions

> _Provided by PR **Assignees**_

1. Create a task using the Create New Task Button. Task should appear under "Active Tasks".
2. Click the checkbox. This should move the task to "Completed Tasks".
3. Create another task using the Create New Task Button. Task should appear under "Active Tasks".
4. Click the task, which pulls up an "Edit Task" modal. Change the task description and click Save. Task description should be changed on its card.
5. Click the task again, pulling up the same modal as before. Click the trash icon. This should delete the task completely.

## 🔎 Reviewer Checklist

> _Checked off by PR **Reviewers**_




---

> ### Roles & Responsibilities
>
> #### 👨‍💻 Assignee
>
> - Initiator of this PR (be sure to set in GitHub UI)
> - Addresses feedback and change requests
> - Merges PR once approved (usually deletes branch unless `develop` or `release`)
>
> #### 👩‍💻 Reviewer
>
> - Invited to review PR by **Assignee** (via GitHub UI)
> - Is expected to complete a review and address followup
